packages/expo-cli/src/commands/build-native/iOSBuilder.ts

```diff
@@ -67,6 +71,60 @@ class iOSBuilder implements Builder {
     this.credentials = await provider.getCredentialsAsync(credentialsSource);
   }
 
+  public async configureProjectAsync(): Promise<void> {
+    // TODO: add simulator flow
+    // assuming we're building for app store
+    if (!this.credentials) {
+      throw new Error('Call ensureCredentialsAsync first!');
```

Comment:
You did not respond to my last comment here
It's if you don't want to handle builds without credentials yet, but throwing an exception here is not correct behavior here. missing credentials filed here is the correct state for the project

Reply:
Well, I haven't replied to your comment but I moved the comment (TODO). 
For now, we're only supporting app store builds which require credentials. This means that the missing credentials field is **not** a correct state of the project.